### PR TITLE
Feature/#13 registrationapi

### PR DESCRIPTION
**Pull request by lkolacz**
please, make this code review after merged data model PR:
https://github.com/unicef/un-partner-portal/pull/24

### CODE REVIEW

File: backend/unpp_api/apps/partner/serializers.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        )
+
+
+class PartnerMemberSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = PartnerMember
+        fields = (
+            'id',
+            #'user',
```
Comment (elyk) [inline backend/unpp_api/apps/partner/serializers.py]: We bringing these back?

File: backend/unpp_api/apps/partner/serializers.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            #'partner',
+            'title',
+            # 'role',
+            # 'status',
+        )
+
+
+class PartnerProfileSerializer(serializers.ModelSerializer):
+
+    # partner = PartnerSerializer(read_only=True)
```
Comment (elyk) [inline backend/unpp_api/apps/partner/serializers.py]: Same comment here..

File: backend/unpp_api/apps/partner/serializers.py (hunk 1 above)
Comment (lkolacz) [inline backend/unpp_api/apps/partner/serializers.py]: @elyk I'm not follow You... Could You be more specific ?

Comment (elyk) [inline backend/unpp_api/apps/partner/serializers.py]: @lkolacz sure - why are they commented out and not removed?

Comment (lkolacz) [inline backend/unpp_api/apps/partner/serializers.py]: removed ... in diff branch ... currently on develop this comments doesn't exists 👍

File: backend/unpp_api/apps/partner/serializers.py (hunk 2 above)
Comment (lkolacz) [inline backend/unpp_api/apps/partner/serializers.py]: done

**Pull request merged**